testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -573,18 +573,20 @@ def test_sklearn_affinity_propagation(self, ens1):
                               "clusters: {0} {1}".format(len(cc1), len(cc2))
 
     def test_ClusterCollection_init(self, cc):
-        assert np.all(cc.clusters[0].elements == [0, 1, 2]) and \
-               np.all(cc.clusters[1].elements == [3, 4   ]) and \
-               np.all(cc.clusters[2].elements == [5, 6, 7]) and \
+        if np.testing.assert_allclose(cc.clusters[0].elements , [0,1,2])and \
```

Comment:
I realise `assert_allclose` was mentioned in the original issue... but why are we using `assert_allclose` on a comparison between ints? Shouldn't this be using `assert_equal`?

Reply:
Agree! Additionally there are many other of theses constructs in the testsuite

```bash
philiploche@Philips-MacBook-Air:~/repos/mdanalysis$ grep -R "np.all(" testsuite/MDAnalysisTests
testsuite/MDAnalysisTests/core/util.py:    assert np.all((relpos >= 0.0) & (relpos < 1.0))
testsuite/MDAnalysisTests/core/test_groups.py:        assert np.all(np.in1d(indices, ag.indices))
testsuite/MDAnalysisTests/core/test_groups.py:        assert not np.all(np.in1d(indices, ag.indices))
testsuite/MDAnalysisTests/core/test_groups.py:        assert np.all(np.in1d(indices, ag.atoms.indices))
testsuite/MDAnalysisTests/core/test_groups.py:        assert not np.all(np.in1d(indices, ag.atoms.indices))
testsuite/MDAnalysisTests/core/test_groups.py:    assert np.all(np.in1d(indices, ag.indices))
testsuite/MDAnalysisTests/analysis/test_encore.py:        assert np.all(cc.clusters[0].elements == [0, 1, 2]) and \
testsuite/MDAnalysisTests/analysis/test_encore.py:               np.all(cc.clusters[1].elements == [3, 4   ]) and \
testsuite/MDAnalysisTests/analysis/test_encore.py:               np.all(cc.clusters[2].elements == [5, 6, 7]) and \
testsuite/MDAnalysisTests/analysis/test_encore.py:        assert np.all(cluster.elements == [0, 1, 2]) and \
testsuite/MDAnalysisTests/analysis/test_encore.py:        assert np.all(cluster.metadata['test'] == metadata), \
testsuite/MDAnalysisTests/coordinates/test_pdb.py:        assert np.all(read_universe.atoms.ids == universe.atoms.ids)
testsuite/MDAnalysisTests/coordinates/test_pdb.py:        assert np.all(read_universe.atoms.ids == universe.atoms.indices + 1)
testsuite/MDAnalysisTests/utils/test_transformations.py:        check = np.all(v >= 0.0) and np.all(v < 1.0)
testsuite/MDAnalysisTests/lib/test_util.py:        if not (np.all(box > 0) and a < 180 and b < 180 and c < 180):
testsuite/MDAnalysisTests/lib/test_util.py:        assert np.all(res == 0)
testsuite/MDAnalysisTests/lib/test_util.py:        assert np.all(res == 0)
testsuite/MDAnalysisTests/lib/test_util.py:        assert np.all(res == 0)
testsuite/MDAnalysisTests/lib/test_util.py:                    if not np.all(res == 0.0):
testsuite/MDAnalysisTests/lib/test_distances.py:        assert np.all((coords < 0.0) | (coords == box[:3]))
testsuite/MDAnalysisTests/lib/test_distances.py:        assert np.all(res >= 0.0)
testsuite/MDAnalysisTests/lib/test_distances.py:        assert np.all(res < box[:3])
testsuite/MDAnalysisTests/lib/test_distances.py:        assert np.all((relcoords < 0.0) | (relcoords == 1.0))
testsuite/MDAnalysisTests/lib/test_distances.py:        assert np.all(relres >= 0.0)
testsuite/MDAnalysisTests/lib/test_distances.py:        assert np.all(relres < 1.0)
```

@aditi2906 will you be able handle them all the same time or should I create a separate issue?

Reply:
@IAlibay I do agree, using assert_equal will be a sensible call since we are dealing with scalars and arrays. I will do the required changes and push the code.

Reply:
@PicoCentauri I will do it within this very issue and commit the code soon!